GridiconsDemo/GridiconsDemo/ViewController.swift

```diff
@@ -69,7 +69,7 @@ extension ViewController: UICollectionViewDataSource {
     func collectionView(_ collectionView: UICollectionView, cellForItemAt indexPath: IndexPath) -> UICollectionViewCell {
```

Comment:
On Line 52 of this class, if you add:

```
stepper.stepValue = value == 18 ? 6 : 12
```

It will also solve issue #3.

Reply:
Thanks :) I think I'll leave this for this PR, but we can always look at solutions in the future.